services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubService.java

```diff
@@ -20,8 +26,7 @@
      * @throws NoSuchRepositoryException If the specified repository does not exist
      * @throws IllegalArgumentException  If the repository name is not specified
      */
-    GitHubRepository fork(String repositoryFullName) throws NoSuchRepositoryException,
-            IllegalArgumentException;
+    GitHubRepository fork(String repositoryFullName);
```

Comment:
An alternative solution to comply with Sonar is to mention these runtime exceptions in the Javadoc. But I'm fine with keeping them in the method signature if you prefer.